test/system/kind.test.ts

```diff
@@ -135,7 +135,7 @@ tap.test('Kubernetes-Monitor with KinD', async (t) => {
   require('../../src');
 
   // TODO: replace with being event driven?
-  // will still need SOME timeout 
+  // will still need SOME timeout
```

Comment:
This change probably shouldn't be part of this commit. Maybe committed by accident since your linting tool has fixed it?
While it does remove an extra space, it's in a comment, so really not that relevant and it doesn't improve the quality of the codebase 😆